.github/workflows/on-pull_request-opened-synchronize-reopened.yml

```diff
@@ -20,6 +20,7 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           ROOT_POM_DIRECTORY: repo/source
+          JDK_VERSION: 21
```

Comment:
Shouldn't we update the action to have 21 by default? @SerhatG

Reply:
Let's talk about this in the technical meeting tomorrow.. :)

Reply:
Based on meeting, the default has now been changed to 21.